js/modules/aioseop_module.js

```diff
@@ -1,345 +1,606 @@
-		if ( !this.checked )
-			jQuery(this).parents('div:eq(0)').find('input[value="all"]:checkbox').prop('checked', this.checked);
-    });
-
-	jQuery(".aioseop_tab:not(:first)").hide();
-    jQuery(".aioseop_tab:first").show();
-    jQuery("a.aioseop_header_tab").click(function(){
-            var stringref = jQuery(this).attr("href").split('#')[1];
-            jQuery('.aioseop_tab:not(#'+stringref+')').hide('slow');
-            jQuery('.aioseop_tab#' + stringref).show('slow');
-            jQuery('.aioseop_header_tab[href!="#'+stringref+'"]').removeClass('active');
-            jQuery('.aioseop_header_tab[href="#' + stringref+'"]').addClass('active');
-            return false;
-    });    
-});
+/**
+ * Controls all the styling of the plugin.
+ *
+ * AIOSEOP Updates class.
+ * @author Michael Torbert.
+ * @author Semper Fi Web Design.
+ * @copyright http://semperplugins.com
+ * @version 1.0.0
+ */
+if ( typeof aiosp_data != 'undefined' ) {
+
+	/**
+	 * @since 1.0.0
+	 * @param int $index.
+	 * @param $value
+	 */
+	jQuery.each( aiosp_data, function( index, value ) {
+//		aiosp_data[index] = value.json.replace(/&quot;/g, '"');
+//		aiosp_data[index] = jQuery.parseJSON( value );
+		if ( index == 0 ) {
+			if ( typeof value.condshow == 'undefined' ) {
+				aiosp_data[ index ].condshow = [];
+			}
+		} else {
+			if ( typeof value.condshow != 'undefined' ) {
+				aiosp_data[ 0 ].condshow =
+					jQuery.merge( aiosp_data[0].condshow, value.condshow );
+			}
+		}
+	});
+	aiosp_data = aiosp_data[0];
+}
+
+/**
+ * @summary Changes visibility.
+ *
+ * @since 1.0.0
+ * @param int $id.
+ */
+function toggleVisibility( id ) {
+	var e = document.getElementById( id );
+	if ( e.style.display == 'block' )
+		e.style.display = 'none';
+	else
+		e.style.display = 'block';
+}
+
+/**
+ * @summary Counts characters.
+ *
+ * @since 1.0.0
+ * @param String $field.
+ * @param Int $cntfield.
+ * @return Mixed.
+ */
+function countChars( field, cntfield ) {
+	var extra = 0;
+	var field_size;
+	if ( ( field.name == 'aiosp_title' )
+		&& ( typeof aiosp_title_extra !== 'undefined' ) ) {
+		extra = aiosp_title_extra;
+	}
+	cntfield.value = field.value.length + extra;
+	if ( typeof field.size != 'undefined' ) {
+		field_size = field.size;
+	} else {
+		field_size = field.rows * field.cols;
+	}
+	if ( field_size < 10 )
+		return;
+	if ( cntfield.value > field_size ) {
+		cntfield.style.color = "#fff";
+		cntfield.style.backgroundColor = "#f00";
+	} else {
+		if ( cntfield.value > ( field_size - 6 ) ) {
+			cntfield.style.color = "#515151";
+			cntfield.style.backgroundColor = "#ff0";
+		} else {
+			cntfield.style.color = "#515151";
+			cntfield.style.backgroundColor = "#eee";
+		}
+	}
+}
+
+/**
+ * @summary Returns the fields value.
+ *
+ * @since 1.0.0
+ * @param String $field.
+ * @return Mixed.
+ */
+function aioseop_get_field_value( field ) {
+	if ( field.length == 0 )

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected { after 'if' condition.](https://www.codacy.com/app/mrtorbert/all-in-one-seo-pack/file/2692638761/issues/source?bid=3283401&fileBranchId=3340169#l93)